### PR TITLE
jbehave steps to send requests asynchronously

### PR DESCRIPTION
**Pull request by rdrozdov-tc**

### CODE REVIEW

File: examples/src/main/java/io/github/tcdl/examples/MSBUtil.java [hunk 1]
```diff
@@ -14,14 +14,24 @@
 import java.util.Map;
 
 /**
- * Created by rdrozdov-tc on 6/15/15.
+ * Utility to simplify using requester and responder server
  */
-public class BaseExample {
+public class MSBUtil {
```
Comment (rkatsyuryna) [inline examples/src/main/java/io/github/tcdl/examples/MSBUtil.java:19]: can we just have MsbUtil but not MSBUtil?  Also name of class is to general, why not ComponentCreatorHelper or simple RequesterCreatorHelper (as we create only Requesters here)?

File: examples/src/main/java/io/github/tcdl/examples/MultipleRequesterResponder.java [hunk 2]
```diff
@@ -11,9 +11,12 @@
 /**
  * Created by anstr on 6/9/2015.
  */
-public class MultipleRequesterResponder extends BaseExample {
+public class MultipleRequesterResponder {
+
     private static final Integer NUMBER_OF_RESPONSES = 1;
 
+    private MSBUtil util = MSBUtil.getInstance();
```
Comment (rkatsyuryna) [inline examples/src/main/java/io/github/tcdl/examples/MultipleRequesterResponder.java:18]: very strange method as for Util class - getInstance . AFAIS utility classes need not instance creation

File: examples/src/test/resources/io/github/tcdl/examples/multiple_requests_to_date_extractor.story [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Lifecycle:
+Before:
+Given init
+And microservice DateExtractor
+After:
+Outcome: ANY
+Given shutdown
+
+Scenario: Sending multiple requests to date extractor microservice in parallel
```
Comment (rkatsyuryna) [inline examples/src/test/resources/io/github/tcdl/examples/multiple_requests_to_date_extractor.story:9]: multiple is how much? can we set the number e.g 5 or 2000?

File: examples/src/main/java/io/github/tcdl/examples/MSBUtil.java [hunk 4]
```diff
@@ -14,14 +14,24 @@
...
 
-    MsbContext context;
+    private static MSBUtil instance;
 
-    public BaseExample() {
-        init();
+    static {
+        instance = new MSBUtil();
```
Comment (vso-tc) [inline examples/src/main/java/io/github/tcdl/examples/MSBUtil.java:24]: Why do you need this static block?

File: examples/src/main/java/io/github/tcdl/examples/MSBUtil.java [hunk 5]
```diff
@@ -14,14 +14,24 @@
...
+    }
+
+    private MsbContext context;
+
+    private MSBUtil() {
+    }
+
+    public static MSBUtil getInstance() {
+        instance.init();
```
Comment (vso-tc) [inline examples/src/main/java/io/github/tcdl/examples/MSBUtil.java:33]: weird code. When you call getInstance you overwrite msb context in the single instance you use

**Pull request closed without merging**